proto/chart/v1/stream.proto

```diff
@@ -64,3 +64,20 @@ message ChartsAndDimensionsUpdated {
   repeated chart.v1.ChartDimensionUpdated dimensions = 2;
   uint64 batch_id = 3;
 }
+
+// RetentionUpdated includes the available retentions (in seconds) of the dimensions - of a specific node instance and memory-mode -
+// on a per update_every level.
+// This message is sent over upon Agent Database rotation events to inform the Cloud in total about the newly updated data retentions
+// of a node instance's dimensions.
+message RetentionUpdated {
```

Comment:
[ultra-minor] Since this is not strictly stream-related (start/stop/reset etc) but more like maintenance-related, we could also use a new file for this.